src/test/util/event.rs

```diff
@@ -68,46 +75,110 @@ impl CommandEvent {
     }
 }
 
-#[derive(Default)]
+#[derive(Clone, Debug)]
```

Comment:
This is part of the work involved in RUST-572, but I needed it now for the two prose tests introduced in `src/sdam/test.rs`. There is some copy pasted code here from the CMAP runner's implementation of this. We can remove the duplicate in RUST-572 and just use the single `EventHandler` then.